ethereum-consensus/src/configs/holesky.rs

```diff
@@ -0,0 +1,96 @@
+use crate::{
+    configs::Config,
+    primitives::{Epoch, ExecutionAddress, Gwei, Version, FAR_FUTURE_EPOCH},
+};
+
+// `2**14` (= 16,384)
+pub const MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: usize = 16384;
+pub const MIN_GENESIS_TIME: u64 = 1694786100;
+// 0x00017000
+pub const GENESIS_FORK_VERSION: Version = [0, 1, 112, 0];
+// Genesis delay 5 seconds
+pub const GENESIS_DELAY: u64 = 300;
+pub const SECONDS_PER_SLOT: u64 = 12;
+pub const SECONDS_PER_ETH1_BLOCK: u64 = 14;
+pub const MIN_VALIDATOR_WITHDRAWABILITY_DELAY: Epoch = 256;
+pub const SHARD_COMMITTEE_PERIOD: Epoch = 256;
+pub const ETH1_FOLLOW_DISTANCE: u64 = 2048;
+pub const EJECTION_BALANCE: Gwei = 16 * 10u64.pow(9);
```

Comment:
https://github.com/eth-clients/holesky/blob/main/custom_config_data/config.yaml#L61

```suggestion
pub const EJECTION_BALANCE: Gwei = 28 * 10u64.pow(9);
```